src/Adapters/AzureManagementAdapter.ts

```diff
@@ -567,4 +571,94 @@ export default class AzureManagementAdapter implements IAzureManagementAdapter {
             return new AzureResourcesData([newRoleAssignment]);
         }, 'azureManagement');
     }
+
+    async getTimeSeriesConnectionInformation(adtInstanceIdentifier: {
+        id?: string;
```

Comment:
is it possible to make this type a bit more strict? At least one of these properties is required but it looks like right now both are optional.

Reply:
added strict type, yeah they are mutually exclusive, only one of them should be provided